scripts/prepare_nerfstudio.py

```diff
@@ -0,0 +1,160 @@
+"""Creates transforms.json file and resized images for running Nerfstudio from gtsfm output.
+
+This contains code from Nerfstudio for parsing data into the Nerfstudio format.
+https://github.com/nerfstudio-project/nerfstudio/blob/a121d76fd085f0fb356abf150c089c42eecbd066/nerfstudio/process_data/colmap_converter_to_nerfstudio_dataset.py#L29
+https://github.com/nerfstudio-project/nerfstudio/blob/a121d76fd085f0fb356abf150c089c42eecbd066/nerfstudio/process_data/colmap_utils.py#L16
+
+Author: Jon Womack
+"""
+import argparse
+import json
+import os
+from enum import Enum
+from pathlib import Path
+from typing import Tuple
+
+import numpy as np
+from gtsam import Rot3
+
+from gtsfm.utils import images
+from gtsfm.utils import io as io_utils
+
+
+class CameraModel(Enum):
+    """Enum for camera types."""
+
+    OPENCV = "OPENCV"
+    OPENCV_FISHEYE = "OPENCV_FISHEYE"
+
+
+CAMERA_MODELS = {
+    "perspective": CameraModel.OPENCV,
+    "fisheye": CameraModel.OPENCV_FISHEYE,
+}
+
+
+def colmap_to_json(data_dir: str, output_dir: str, camera_model: CameraModel) -> Tuple[int, int]:
+    """Converts GTSfM's cameras.txt to to a JSON file, saves it in output_dir.
+    Args:
+        cameras_path: Path to the cameras.txt file.
+        images_path: Path to the images.txt file.
+        output_dir: Path to the output directory.
+        camera_model: Camera model used.
+    Returns:
```

Comment:
nit: newlines between L37 and L38, and between L42 and L43. Can we note in the return args section "Image dimensions as a tuple (width, height)."